.github/workflows/pull_request.yml

```diff
@@ -182,7 +182,7 @@ jobs:
       - name: Make Deps
         run: |
           python -m pip install --upgrade pip
-          pip install bandit
+          pip install "importlib-metadata<5" bandit
```

Comment:
Not related to this PR really, but a new release of importlib-metadata happened to come out and break bandit.  This fixes that so the actions can pass.